deck.gl__core/index.d.ts

```diff
@@ -1110,6 +1110,8 @@ declare module "@deck.gl/core/lib/layer" {
 		(o: PickInfo<D>, e: HammerInput): any;
 	}
 	export type DataSet<D> = Iterable<D>;
+	export type WidthUnits = "meters" | "pixels";
```

Comment:
Unfortunately, there's no util that handles width units. As a result, I think it's logical for this to live in the layer module. I toyed with the idea of importing and re-exporting the type in `CompositeLayer` as well but didn't want to overreach.